doc/lua_api.md

```diff
@@ -7464,6 +7464,15 @@ use the provided load and write functions for this.
 * `from_file(filename)`: Experimental. Like `from_string()`, but reads the data
   from a file.
 
+`AsyncJob`
+----------
+An `AsyncJob` is a reference to a job to be run in an async environment.
+
+### Methods
+* `get_id()`: get the numeric ID of the job.
```

Comment:
ah I forgot, I removed this one too
do you think it's useful for modders?

Reply:
I am not sure.

Is it intended to (in the future) implement directly passing messages to async jobs? My anticipation of this (and that jobs would _eventually_ be implemented as userdata objects one way or another) was the main reason for a `get_id()` method.

Aside from that, a getter would make sense to make it explicit that modders should not change `job.id` directly, although it is also possible to just mention this in the documentation instead of having a getter.

Reply:
My personal opinion is that async jobs should be used to offload relatively short and finite tasks from the main thread. (reason being the finite worker thread count)
Therefore IPC that works specifically with async jobs does not make sense.

Reply:
Ok. I assume I will just remove this then.